src/H5Cimage.c

```diff
@@ -2650,10 +2662,12 @@ static herr_t
 H5C__prep_for_file_close__setup_image_entries_array(H5C_t *cache_ptr)
 {
     H5C_cache_entry_t *entry_ptr;
-    H5C_image_entry_t *image_entries   = NULL;
-    uint32_t           entries_visited = 0;
-    unsigned           u;                   /* Local index variable */
-    herr_t             ret_value = SUCCEED; /* Return value */
+    H5C_image_entry_t *image_entries = NULL;
+#ifndef NDEBUG
+    uint32_t entries_visited = 0;
+#endif                            /* NDEBUG */
```

Comment:
Also, tail comments for ifdefs that are one line above just turn into malformed noise, as is the case here. They are really only useful for stuff that rolls off the screen.

Reply:
OK, I'll update those